cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -1,20 +1,108 @@
+import re
+import warnings
 from datetime import datetime
-from typing import ClassVar
+from enum import StrEnum
+from typing import Any, ClassVar
 
+from pydantic import Field, HttpUrl, TypeAdapter, field_validator
 from rdflib import Namespace
 
-from .base import Prefix, RoleTypes, RuleModel
-from .domain_rules import DomainMetadata
+from cognite.neat.rules import exceptions
+from cognite.neat.rules.models.rdfpath import (
+    AllReferences,
+    Hop,
+    RawLookup,
+    SingleProperty,
+    SPARQLQuery,
+    TransformationRuleType,
+    Traversal,
+)
 
+from .base import (
+    Prefix,
+    RoleTypes,
+    RuleModel,
+    SheetList,
+    prefix_compliance_regex,
+    version_compliance_regex,
+)
+from .domain_rules import DomainClass, DomainMetadata, DomainProperty
 
-class InformationArchitectMetadata(DomainMetadata):
+
+class MatchType(StrEnum):
+    exact = "exact"
+    partial = "partial"
+
+
+class InformationMetadata(DomainMetadata):
```

Comment:
Suggest we inherit from the base metadata and not domain.

We do not want the creator on metadata.
```suggestion
class InformationMetadata(BaseMetadata):
```